docs/variables.rst

```diff
@@ -134,4 +134,4 @@ Example playbook that uses the extra_vars:
 Using vaulted strings for variables
 ------------------------------------
 
-The variables file can contain vaulted strings. Refer to the `Rulebooks <rulebooks.html>`_ page for more information.
\ No newline at end of file
+The variables file can contain vaulted strings. Refer to the `Rulebooks <rulebooks.rst>`_ page for more information.
```

Comment:
These links are correct. Are intended to be used for the compiled version: 
https://ansible--759.org.readthedocs.build/projects/rulebook/en/759/rulebooks.rst

Reply:
oh...sorry..

Reply:
np! thanks for the contribution anyway :)